src/vsr/replica.zig

```diff
@@ -4982,24 +4982,32 @@ pub fn ReplicaType(
             assert(self.primary());
 
             if (message.header.release.value < self.release_client_min.value) {
-                log.warn("{}: on_request: ignoring invalid version (client={} version={}<{})", .{
+                log.warn("{}: on_request: ignoring invalid client version; too low" ++
+                    " (client={} version={}<{})", .{
                     self.replica,
                     message.header.client,
                     message.header.release,
                     self.release_client_min,
                 });
-                self.send_eviction_message_to_client(message.header.client, .release_too_low);
+                self.send_eviction_message_to_client(
+                    message.header.client,
+                    .client_release_too_low,
+                );
                 return true;
             }
 
             if (message.header.release.value > self.release.value) {
-                log.warn("{}: on_request: ignoring invalid version (client={} version={}>{})", .{
+                log.warn("{}: on_request: ignoring invalid client version; too high " ++
+                    "(client={} version={}>{})", .{
```

Comment:
Maybe `s/invalid/unsupported`? The version _is_ valid, but we chose not to support it.